tmt/steps/execute/scripts/tmt-report-result

```diff
@@ -94,7 +94,7 @@ write_report_file () {
 
 declare -A RESULTS_HIERARCHY=( ["FAIL"]=4 ["WARN"]=3 ["PASS"]=2 ["SKIP"]=1 )
 # Create the output file in which to store the results.
-REPORT_RESULT_OUTPUTFILE="$TMT_TEST_DATA/reportResult"
+REPORT_RESULT_OUTPUTFILE="$TMT_TEST_DATA/report-result"
```

Comment:
Could it be called 'rstrnt-result' or similar? I'm afraid of conflict or too similar name for #1074)

Reply:
Makes sense, this feature is very closely related to `restraint` so we should include it in the name.